### PR TITLE
Update survey-type.ttl

### PR DESCRIPTION
**Pull request by KellyVance**
Addition of notations

### CODE REVIEW

File: vocabularies/survey-type.ttl [hunk 1]
```diff
@@ -202,17 +215,20 @@ svytp:gravity-gradiometry a skos:Concept ;
     skos:broader svytp:gravity ;
     skos:definition "A variation of gravity survey where the gradient of the gravity field is measured instead of the field directly."@en ;
     skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
+    skos:notation "at" ;gy
```
Comment (LukeHauck) [inline vocabularies/survey-type.ttl]: remove _gy_ at end. Otherwise looks good.

Comment (KellyVance) [reply vocabularies/survey-type.ttl]: fixed to
skos:notation "gy" ;

File: vocabularies/survey-type.ttl [hunk 2]
```diff
@@ -165,12 +174,14 @@ svytp:process a skos:Concept ;
...
+    skos:notation "ps" ;
     skos:prefLabel "Process"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
 
 svytp:atmospheric a skos:Concept ;
     skos:definition "A survey measuring chemical and physical properties of the atmosphere."@en ;
     skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
+    skos:notation "at" ;
     skos:prefLabel "Atmospheric"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
```
Comment (GSQ-AI) [inline vocabularies/survey-type.ttl]: Are these electrical altLabels appropriate, or should they be methods instead?
@KellyVance @dxwell @DavidCrosswellGSQ

Comment (KellyVance) [reply vocabularies/survey-type.ttl]: yeah. I think they are methods. should migrate them soon

File: vocabularies/survey-type.ttl [hunk 3]
```diff
@@ -241,25 +260,29 @@ svytp:radiometric a skos:Concept ;
...
+    skos:notation "sp" ;
     skos:prefLabel "Spectral"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
 
 svytp:gravity a skos:Concept ;
     skos:definition "A survey where the variation of the Earth's gravity field in measured to determine the distribution of density in the region"@en ;
     skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
+    skos:notation "gv" ;
     skos:prefLabel "Gravity"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
```
Comment (GSQ-AI) [inline vocabularies/survey-type.ttl]: Should we include a new concept for "Spatial" surveys? 
Stuff like stereophotogrammetry, Shuttle Radar Topography Mapping, LIDAR, Theodolite surveying etc?
@dxwell @DavidCrosswellGSQ @KellyVance @geoderekh @Greenwoodmatthew 

e.g.
```
svytp:spatial a skos:Concept ;



    skos:definition "A survey measuring the spatial characteristics of an object or region."@en ;



    skos:inScheme <http://linked.data.gov.au/def/survey-type> ;



    skos:notation "st" ;



    skos:prefLabel "Spatial"@en ;



    skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
```

File: vocabularies/survey-type.ttl [hunk 4]
```diff
@@ -241,25 +260,29 @@ svytp:radiometric a skos:Concept ;
         "Gamma-Ray Spectrography"@en ;
```
Comment (GSQ-AI) [inline vocabularies/survey-type.ttl]: Survey method not altabels? Maybe?

File: vocabularies/survey-type.ttl (hunk 3 above)
Comment (Greenwoodmatthew) [reply vocabularies/survey-type.ttl]: I dont know if spatial is the right term. 'Elevation' survey 'digital terrain model' survey?

Comment (KellyVance) [reply vocabularies/survey-type.ttl]: @GSQ-AI Good idea. Added.

@Greenwoodmatthew I think 'spatial' can encompass any survey that defines the shape of stuff. So topographic surveys, point-cloud surveys (eg. ground surveys, underground mine, or object rendering). 

Things like Topographic could be a narrower terms if-and-when we need that granularity with Digital Elevation, Digital Terrain, Elevation as altLabels

**Pull request merged**